src/main/java/econovation/moongtaengi/study/domain/StudyMember.java

```diff
@@ -0,0 +1,37 @@
+package econovation.moongtaengi.study.domain;
+
+import econovation.moongtaengi.global.entity.BaseEntity;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Table(name = "study_members")
+public class StudyMember extends BaseEntity {
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "study_id")
+    private Study study;
+
+    @Column(nullable = false)
+    private Long memberId;
+
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = false)
+    private StudyRole role;
+
+    public StudyMember(Study study, Long memberId, StudyRole role) {
+        this.study = study;
+        this.memberId = memberId;
+        this.role = role;
+    }
+}
```

Comment:
```suggestion
@Entity
@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@Table(name = "study_members")
public class StudyMember extends BaseEntity {

    @Column(nullable = false)
    private Long studyId;  // FK를 직접 ID로 관리

    @Column(nullable = false)
    private Long memberId;

    @Enumerated(EnumType.STRING)
    @Column(nullable = false)
    private StudyRole role;

    public StudyMember(Long studyId, Long memberId, StudyRole role) {
        this.studyId = studyId;
        this.memberId = memberId;
        this.role = role;
    }
}
```

위에 Study 부분에 제안했던 코드랑 이것도 같이 참고해보면 좋을거 같아요
불필요한 관계를 줄이기 위해서 이런 방식을 생각해봤어요.